active_directory_ldap/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: active_directory_ldap
 title: Active Directory LDAP
 description: This plugin utilizes Microsoft\'s Active Directory service to create and manage domains, users, and objects within a network
```

Comment:
Let's fix this why we're here. I believe the SDK now handles this and you can remove the escape

<img width="321" alt="Screen Shot 2020-10-07 at 11 48 35 AM" src="https://user-images.githubusercontent.com/30870727/95355057-204c0600-0893-11eb-91b4-bf69990012e6.png">

Reply:
@mhofert-r7 still need to tackle this one